src/playground/services/invoices/retrieve/endpoint.clj

```diff
@@ -20,8 +20,15 @@
         record (->> (logic/to-query id)
                     (h/format)
                     (jdbc/query db)
-                    (first))]
-    (if record
+                    (first))
+        all-records (->> (logic/all-query)
+                         (h/format)
+                         (jdbc/query db))
+        ]
+    (if (= id 0)
```

Comment:
I know it's not a super cool method to have to if's... 
Since `0` is not used by Postgres in the SERIAL and passes the spec, I thought it could be used to return 
`SELECT * FROM users;`